.gitlab-ci.yml

```diff
@@ -23,3 +24,54 @@ build-composer-dependencies:
     paths:
       - .composer
       - .Build
+
+php-lint-php7.4:
+  stage: lint
+  script:
+    - composer ci:php:lint
+  allow_failure: true
```

Comment:
I think for readability/consistency, explicitly (and redundantly) using the 7.4 image here would be helpful.